typings/internalBinding.d.ts

```diff
@@ -439,3 +439,262 @@ declare function InternalBinding(binding: 'config'): {
   bits: number,
   hasDtrace: boolean
 }
+
+declare namespace InternalFSBinding {
```

Comment:
This file is already quite long. I'm thinking about moving this to a new one called `internalBinding.fs.d.ts`.

Reply:
sounds good but I would avoid the .fs, just use a subdirectory or _ , we don't use `.` as separators in core files to my knowledge, they are only for extensions